### PR TITLE
Mehrdad dashboard

### PR DESCRIPTION
**Pull request by mehrdadhalali**
The temperature now gets filtered **before** the charts are created. Also, the line chart no longer starts from 0.

### CODE REVIEW

File: dashboard/transform_dashboard.py [hunk 1]
```diff
@@ -74,18 +74,19 @@ def create_temperature_line(df: pd.DataFrame) -> alt.Chart:
 
     df_filtered = df[df['at'] >= two_hours_ago]
 
-    title = alt.TitleParams('Plant temperature over time', anchor='middle')
+    title = alt.TitleParams(
```
Comment (Leon-JS02) [inline dashboard/transform_dashboard.py:85]: This is a better title

File: dashboard/transform_dashboard.py [hunk 2]
```diff
@@ -74,18 +74,19 @@ def create_temperature_line(df: pd.DataFrame) -> alt.Chart:
...
+        'Plant temperature fluctuation over time', anchor='middle')
     temp_line_chart = alt.Chart(df_filtered, title=title).mark_line().encode(
         x='at:T',
-        y='temperature:Q',
+        y=alt.Y('temperature:Q').scale(zero=False),
```
Comment (Leon-JS02) [inline dashboard/transform_dashboard.py:89]: This will improve the focus of the visualisation! Nice

**Review by Leon-JS02 — APPROVED** [review]

File: dashboard/transform_dashboard.py [hunk 3]
```diff
@@ -40,10 +40,12 @@ def create_botanist_pie(df: pd.DataFrame) -> alt.Chart:
     grouped_df["botanist_name"] = grouped_df["botanist_id"].map(
         botanist_mapping)
 
-    title = alt.TitleParams('Botanist plant count', anchor='middle')
+    title = alt.TitleParams('Recordings taken per botanist', anchor='middle')
     botanist_pie_chart = alt.Chart(grouped_df, title=title).mark_arc().encode(
         theta="plant_id:Q",
-        color="botanist_name:N"
+        color="botanist_name:N",
+        tooltip=[alt.Tooltip(field="botanist_name", title="Botanist Name"),
```
Comment (YakMan101) [inline dashboard/transform_dashboard.py:47]: Nice clarification of hints using tooltip.

File: dashboard/transform_dashboard.py [hunk 4]
```diff
@@ -54,12 +56,18 @@ def create_temperature_bar(df: pd.DataFrame) -> alt.Chart:
 
     df['at'] = pd.to_datetime(df['at'])
 
+    df["high_temperature"] = df["temperature"] >= 50
```
Comment (YakMan101) [inline dashboard/transform_dashboard.py:59]: Nice way to warn dashboard user.

**Review by YakMan101 — APPROVED** [review]

**Pull request merged**